pkg/subctl/verify/connectivity/run.go

```diff
@@ -24,6 +24,6 @@ import (
 	_ "github.com/submariner-io/submariner/test/e2e/framework"
```

Comment:
let's remove those two files, connectivity/run and discovery/run. we just need to call shipyard with the right focus regex :)

Reply:
Agree. testing changes locally to make sure it works, will push next.